src/Libt8.jl

```diff
@@ -17048,61 +18425,35 @@ const P4EST_ENABLE_VTK_BINARY = 1
 
 const P4EST_ENABLE_VTK_COMPRESSION = 1
 
-const P4EST_HAVE_ALIGNED_ALLOC = 1
-
-const P4EST_HAVE_GNU_QSORT_R = 1
-
-const P4EST_HAVE_MATH = 1
-
-const P4EST_HAVE_POSIX_MEMALIGN = 1
+const P4EST_HAVE_FSYNC = 1
 
 const P4EST_HAVE_ZLIB = 1
 
-const P4EST_LDFLAGS = "-L/workspace/destdir/lib"
-
-const P4EST_LIBS = "-lz -lm "
-
-const P4EST_LT_OBJDIR = ".libs/"
-
-const P4EST_MEMALIGN = 1
-
-const P4EST_SIZEOF_VOID_P = 8
+const P4EST_LDFLAGS = "-Wl,-rpath -Wl,/workspace/destdir/lib -Wl,--enable-new-dtags -L/workspace/x86_64-linux-gnu-libgfortran5-cxx11-mpi+mpich/destdir/lib"
 
-const P4EST_MEMALIGN_BYTES = P4EST_SIZEOF_VOID_P
-
-const P4EST_MPI = 1
-
-const P4EST_MPIIO = 1
+const P4EST_LIBS = "   m"
 
 const P4EST_PACKAGE = "p4est"
 
 const P4EST_PACKAGE_BUGREPORT = "p4est@ins.uni-bonn.de"
 
 const P4EST_PACKAGE_NAME = "p4est"
 
-const P4EST_PACKAGE_STRING = "p4est 2.8.5.367-931f"
+const P4EST_PACKAGE_STRING = "p4est 0.0.0"
```

Comment:
Is that how it's supposed to be?

Reply:
Of course not. This seems to be an problem in the CMake build system upstream and should be fixed there. The definition in the header files says indeed "0.0.0". Since p4est is a helper library in not supposed to be used by the users directly and this variable is used nowhere, I suggest to leave it that way.

Reply:
I opened an issue about that: https://github.com/DLR-AMR/t8code/issues/1294

Reply:
Ok.